### PR TITLE
Add option to toggle display with the scoreboard

### LINKED ISSUE #1: Self dissappearing Deja-vu or keybind?
hello there, Is there any way to make plugin hide after some time like 15 secs after match starts? It's kinda annoying to stay there as the whole match or a keybind to enable/disable it?

### PR DESCRIPTION
**Pull request by BellyandKen**
Added the option to "Toggle with scoreboard". 

Fixes #1

### CODE REVIEW

File: DejaVu.cpp [hunk 1]
```diff
@@ -172,7 +172,8 @@ void DejaVu::onLoad()
 	RegisterCVar("cl_dejavu_show_metcount", "Show the met count instead of your record", true, this->showMetCount);
 
 	RegisterCVar("cl_dejavu_visuals", "Enables visuals", true, this->enabledVisuals);
-
+	RegisterCVar("cl_dejavu_toggle_with_scoreboard", "Toggle with scoreboard (instead of always on)", true, this->toggleWithScoreboard);
+	//can't get it showing yet
```
Comment (adamk33n3r) [inline DejaVu.cpp:176]: Is this comment necessary anymore?

File: DejaVu.cpp [hunk 2]
```diff
@@ -172,7 +172,8 @@ void DejaVu::onLoad()
 	RegisterCVar("cl_dejavu_show_metcount", "Show the met count instead of your record", true, this->showMetCount);
 
 	RegisterCVar("cl_dejavu_visuals", "Enables visuals", true, this->enabledVisuals);
-
+	RegisterCVar("cl_dejavu_toggle_with_scoreboard", "Toggle with scoreboard (instead of always on)", true, this->toggleWithScoreboard);
```
Comment (adamk33n3r) [inline DejaVu.cpp:175]: What are your thoughts on the default here? You have it set to `true` but I'm guessing that was just a copy/paste artifact.

**Review by adamk33n3r — CHANGES_REQUESTED** [review]

File: DejaVu.cpp (hunk 1 above)
Comment (BellyandKen) [reply DejaVu.cpp:176]: Haha nope

File: DejaVu.cpp (hunk 2 above)
Comment (BellyandKen) [reply DejaVu.cpp:175]: Yes, this was a copy/paste thing. I do think default should be what yours has always been!

**Review by adamk33n3r — APPROVED** [review]

**Pull request merged**